js/app.js

```diff
@@ -1,205 +0,0 @@
-    deleteList.call(this);
-  }));
-  $('#listNameForm').submit(function(e) {
-    var name = $('#listName').val();
-    if (name === '') {
-      $('#listName').addClass('error');
-    } else {
-      $('#listName').removeClass('error');
-      addListName(name);
-      var shopLists = getLists();
-      shopLists.push(name);
-      setLists('Shopping Lists', shopLists);
-      var emptyList = [''];
-      setLists(name.concat('.Checked'), emptyList);
-      $('.listItems ul').empty();
-    }
-    e.preventDefault();
-  });
-  $('#itemNameForm').submit(function(e) {
-    var name = $('#itemName').val();
-    if (name === '') {
-      $('#itemName').addClass('error');
-    } else {
-      $('#itemName').removeClass('error');
-      addItem(name);
-      var listItems = getListsItems($('.listNames ul').find("input:radio:checked").val());
-      if (listItems === null || listItems[0] === '')
-        listItems = [name];
-      else
-        listItems.push(name);
-      setListItem($('.listNames ul').find("input:radio:checked").val(), listItems);
-    }
-    e.preventDefault();
-  });
-  $('.listNames ul').click(function() {
-    $('.listItems ul').empty();
-    var listItems = getListsItems($('.listNames ul').find("input:radio:checked").val());
-    if (listItems !== null && listItems[0] !== '') {
-      parseList(listItems, addItem);
-      var listItems2 = getListsItems($('.listNames ul').find("input:radio:checked").val().concat('.Checked'));
-      parseChecked(listItems2);
-    }
-  });
-  $('ul').on('click', 'li', function(){
-    if ($(this).find('input').is(':radio')) {
-      $('li').each(function() {
-        $(this).toggleClass('highlight', false);
-      });
-    }
-    console.log($(this).find("input:radio").val());
-    if ($(this).find("input:radio").val()){
-      if (!$(this).find('input').prop('checked')){
-        $(this).find('input').prop('checked', true);
-      }
-      $(this).toggleClass('highlight', this.checked);
-    } else {
-      var checked = getListsItems($('.listNames ul').find("input:radio:checked").val().concat('.Checked'));
-      if ($(this).find('input').prop('checked')){
-        $(this).find('input').prop('checked', false);
-        checked[$(this).index()] = '';
-      } else {
-        $(this).find('input').prop('checked', true);
-        checked[$(this).index()] = 'checked';
-      }
-      $(this).toggleClass('highlight', this.checked);
-      setListItem($('.listNames ul').find("input:radio:checked").val().concat('.Checked'), checked);
-    }
-  });
-});
-function deleteList() {
-  if ($(this).find('input').is(':radio')) {
-    $(this).parent().each(function() {
-      $(this).toggleClass('highlight', false);
-    });
-  }
-  if ($(this).prev().find('input').is(':radio'))
-    deleteListHelper1.call(this);
-  else
-    deleteListHelper2.call(this);
-  if ($(this).prev().find('input').prop('checked') && $(this).prev().find('input').is(':radio') &&
-  $(this).parent().prev().find('input').length === 1){
```

Comment:
Indentation fix not needed for app.js since everything was moved to index.js, ensured indentation was fixed in index.js.